apps/ui/src/helpers/quorum.ts

```diff
@@ -1,24 +1,41 @@
 import { offchainNetworks } from '@/networks';
-import { NetworkID, Proposal } from '@/types';
+import { NetworkID, Proposal, VoteType } from '@/types';
 
 const REJECTION_QUORUM_CHOICE_INDEX = 1;
+const SCORES_FOR_INDEX = 0;
+const SCORES_AGAINST_INDEX = 1;
+const SCORES_ABSTAIN_INDEX = 2;
 
 export function getProposalCurrentQuorum(
   networkId: NetworkID,
   proposal: {
     scores: number[];
+    type: VoteType;
     scores_total: number;
     quorum_type?: string;
+    hide_abstain?: boolean;
   }
 ) {
   if (offchainNetworks.includes(networkId)) {
-    return proposal.quorum_type === 'rejection'
-      ? Number(proposal.scores[REJECTION_QUORUM_CHOICE_INDEX] ?? 0)
-      : Number(proposal.scores_total);
+    if (proposal.quorum_type === 'rejection') {
+      return proposal.scores[REJECTION_QUORUM_CHOICE_INDEX] ?? 0;
+    }
+
+    if (proposal.hide_abstain && proposal.type === 'basic') {
+      return (
+        proposal.scores[SCORES_FOR_INDEX] +
+        proposal.scores[SCORES_AGAINST_INDEX]
+      );
+    }
+
+    return proposal.scores_total;
```

Comment:
I think I'd keep Number conversions, our types are inaccurate (we get strings/bigints/numbers in them) 😆